opack/opack_construct.py

```diff
@@ -0,0 +1,91 @@
+from construct import Adapter, Array, Computed, Filter, Float32l, Float64l, GreedyBytes, GreedyString, Int8ub, Int8ul, \
+    Int16ub, Int32ub, Int32ul, Int64ub, Int64ul, LazyBound, PascalString, Prefixed, RepeatUntil, Struct, Switch, \
+    Timestamp, this
+
+from opack.object_types import OBJ_TYPE_MAP, OPackObjectType, TerminatorObject
```

Comment:
seems like only these objects are supposed to be the exported API from `object_types`. lets prefix all others with `_`

Reply:
fixed